frontend/src/api/generated/default/default.msw.ts

```diff
@@ -0,0 +1,243 @@
+
+import { HttpResponse, delay, http } from "msw";
+import type { RequestHandlerOptions } from "msw";
+
+import type {
+  TagListResponse,
+  TagSelection,
+  UserResponse,
+} from "../gitFeedAPI.schemas";
+
+export const getGetTagsResponseMock = (
+  overrideResponse: Partial<TagListResponse> = {},
+): TagListResponse => ({
+  items: Array.from(
+    { length: faker.number.int({ min: 1, max: 10 }) },
+    (_, i) => i + 1,
+  ).map(() => ({
+    id: faker.number.int({ min: undefined, max: undefined }),
+    name: faker.string.alpha({ length: { min: 10, max: 20 } }),
+    popularityRank: faker.number.int({ min: undefined, max: undefined }),
+  })),
+  totalPages: faker.number.int({ min: undefined, max: undefined }),
+  ...overrideResponse,
+});
+
+export const getGetUsersMeTagSelectionsResponseMock = (
+  overrideResponse: Partial<TagSelection> = {},
+): TagSelection => ({
+  tagIds: Array.from(
+    { length: faker.number.int({ min: 1, max: 10 }) },
+    (_, i) => i + 1,
+  ).map(() => faker.number.int({ min: undefined, max: undefined })),
+  ...overrideResponse,
+});
+
+export const getPutUsersMeTagSelectionsResponseMock = (
+  overrideResponse: Partial<TagSelection> = {},
+): TagSelection => ({
+  tagIds: Array.from(
+    { length: faker.number.int({ min: 1, max: 10 }) },
+    (_, i) => i + 1,
+  ).map(() => faker.number.int({ min: undefined, max: undefined })),
+  ...overrideResponse,
+});
+
+export const getPostAuthGithubCallbackResponseMock = (
+  overrideResponse: Partial<UserResponse> = {},
+): UserResponse => ({
+  user: {
+    id: faker.number.int({ min: undefined, max: undefined }),
+    login: faker.string.alpha({ length: { min: 10, max: 20 } }),
+  },
+  created: faker.helpers.arrayElement([faker.datatype.boolean(), undefined]),
+  ...overrideResponse,
+});
+
+export const getPostAuthRefreshResponseMock = (
+  overrideResponse: Partial<UserResponse> = {},
+): UserResponse => ({
+  user: {
+    id: faker.number.int({ min: undefined, max: undefined }),
+    login: faker.string.alpha({ length: { min: 10, max: 20 } }),
+  },
+  created: faker.helpers.arrayElement([faker.datatype.boolean(), undefined]),
+  ...overrideResponse,
+});
+
+export const getGetTagsMockHandler = (
+  overrideResponse?:
+    | TagListResponse
+    | ((
+        info: Parameters<Parameters<typeof http.get>[1]>[0],
+      ) => Promise<TagListResponse> | TagListResponse),
+  options?: RequestHandlerOptions,
+) => {
+  return http.get(
+    "*/tags",
+    async (info) => {
+      await delay(1000);
+
+      return new HttpResponse(
+        JSON.stringify(
+          overrideResponse !== undefined
+            ? typeof overrideResponse === "function"
+              ? await overrideResponse(info)
+              : overrideResponse
+            : getGetTagsResponseMock(),
+        ),
+        { status: 200, headers: { "Content-Type": "application/json" } },
+      );
+    },
+    options,
+  );
+};
+
+export const getGetUsersMeTagSelectionsMockHandler = (
```

Comment:
これget重なるのどうにかならない？
```suggestion
export const createGetUsersMeTagSelectionsMockHandler = (
```
とか

Reply:
APIのgetと他のとが混じるのは結構混乱の元な気がする